src/app/modules/registration/components/set-location-in-map/set-location-in-map.component.ts

```diff
@@ -428,4 +448,14 @@ export class SetLocationInMapComponent implements OnInit, OnDestroy {
     // Workaround is to set minutes to 59.
     return moment().minutes(59).toISOString(true);
   }
+
+  getTranslatedAccuracies(): SelectOption[] {
+    return [
```

Comment:
Når jeg tester med norsk bokmål, får jeg de hardkodede engelske tekstene i drop-down. Ser du har lagt ved oversettelser, men ser ikke ut som de brukes.

Reply:
Hva bruker du for linter?

Reply:
Glemte å legge inn oversettelsefunksjonen her :O